l10n-pocket/en/pocket/about.ftl

```diff
@@ -0,0 +1,56 @@
+
+pocket-about-find-hidden-gems = Find hidden gems
+
+pocket-about-internet-filled-with-buried-treasures = The internet is filled with buried treasures and our vibrant, well-read community has a real knack for finding them. Through the act of saving and consuming content, Pocket users signal where fascination can be found.
+
+
+# Variables:
+#   $pockethome (url) link to https://getpocket.com/
+#   $pockethomedomain (string) GetPocket.com
+#   $pocketandfirefox (url) https://getpocket.com/pocket-and-firefox
+#   $pocketdailynewsletter (url) https://getpocket.com/explore/pocket-hits-signup
+#   $addtopocket (url) link to https://getpocket.com/add
+pocket-about-our-curators-vet-the-most-fascinating = Our curators vet the most fascinating finds to ensure they’re from a diverse range of publications with a track record of trustworthy and accurate coverage, and we share them on <a id="home-page" href="{ $pockethome }">{ $pockethomedomain }</a>, <a id="pocket-and-firefox" href="{ $pocketandfirefox }">{ -brand-name-pocket } for { -brand-name-firefox }</a>, in our <a id="daily-newsletter" href="{ $pocketdailynewsletter }">daily newsletter</a>, and within the <a id="bottom-pocket-app" href="{ $addtopocket }">{ -brand-name-pocket } app</a>. Our recommendations are available primarily in English and German, though we look forward to sharing in more languages in the future.
+
+
+pocket-about-use-pocket-your-way = Use { -brand-name-pocket } your way
+
+pocket-about-there-are-lots-of-ways = There are lots of ways you can use { -brand-name-pocket }. Some people use it to save stories throughout the day to read when they’re ready. Hunter-gatherer types use it to save all the things they want to read, watch, listen to, or buy. Others use { -brand-name-pocket } to highlight key passages to help them grow, learn, and thrive.
+
+pocket-about-some-folks-love-exploring = And some folks love exploring { -brand-name-pocket }’s recommendations to find inspiration, challenge their views, and see the occasional cat (or dog) story.
+
+pocket-about-know-its-good = Know it's good
+pocket-about-internets-only-as-good-as = The internet’s only as good as the content you encounter, so — by saving and consuming with intent — together we make { -brand-name-pocket } a brighter corner of the internet. And, as part of the { -brand-name-mozilla } and { -brand-name-firefox } family, we do all this with the same dedication to privacy you’ve come to expect from { -brand-name-firefox } and { -brand-name-mozilla }.
+
+pocket-about-welcome-to-the-sunny-side = Welcome to the sunny side of the web.
+pocket-about-start-using-pocket = Start using Pocket
+
+
+# Variables:
+#   $pressemail (email) press@getpocket.com
+pocket-about-contact-us-press = Contact us for press inquiries at <a href="mailto:{ $pressemail }">{ $pressemail }</a>.
```

Comment:
Suggest keeping `mailto:` out of localized content as it's required for link functionality.
```suggestion
#   $pressemail (email) mailto:press@getpocket.com
pocket-about-contact-us-press = Contact us for press inquiries at <a href="{ $pressemail }">{ $pressemail }</a>.
```